### PR TITLE
Working docker builds

### PR DESCRIPTION
**Pull request by deweysasser**
Methodology for specifying Jenkins agent had to change -- it appears that
specifying the node at the stop of stages interferes with overriding nodes in
subsequent stages.

This commit also eliminiates a seperate maven dependencies step, as it now runs
in a seperate agent, which destroys the utility.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
-                sh 'mvn dependency:resolve '
-            }
-        }
         stage('maven build') {
+    agent { docker 'maven:3.3.9' }
             steps {
                 sh 'mvn clean install '
+//		script {
+// 		   stash '*/target/*.jar,*/target/*.war'
+//		}
```
Comment (AndyM84) [inline Jenkinsfile]: Since we don't squash merge, you can delete comments as long as they've made it into a commit once.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,27 +1,38 @@
...
     triggers { pollSCM('H/3 * * * *') }
 
     stages {
-        stage('dependencies') {
-            steps {
-                sh 'mvn dependency:resolve '
-            }
-        }
         stage('maven build') {
+    agent { docker 'maven:3.3.9' }
```
Comment (AndyM84) [inline Jenkinsfile]: Any reason for the formatting change here?

Comment (deweysasser) [inline Jenkinsfile]: Nope -- fixed in subsequent commit.  Should I squash the fix in insead?

**Comment by deweysasser** [comment]
Switching docker pipeline strategies, so canceling this PR.

**Pull request closed without merging**